src/components/AccomodationDetails.css

```diff
@@ -0,0 +1,100 @@
+.hero-image {
+    background-image: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)), url("../img/accImg1.png") !important;
+    height: 627px !important;
```

Comment:
Be careful when using `!important`, it can cause frustrations in styling because sometimes you're not sure if this overrides something else. Only use it when it's really necessary